### DISCUSSION TITLE
OpenSAFELY with a remote desktop environment

### DISCUSSION THREAD

**Discussion by eiliskeeble** [opensafely/documentation#972] [Installing OpenSAFELY]
Question from IT support in my organisation: Is it possible to set up OpenSAFELY from within a remote desktop environment? 

Docker is the predicted barrier to this but wondered if there are others? And if anyone has tried using it in this way?

**Comment by iaindillingham** [comment]
Thanks for your question, @eiliskeeble. I've not set up remote desktop, but assuming you're connecting a guest computer to a host computer, with OpenSAFELY running on the host computer, then would first check that Docker is usable from the guest computer. The host computer will also need to connect to GitHub's container registry, which is where the OpenSAFELY Docker images are stored.

I'm not aware of anyone taking this approach. However, our [*Getting Started*](https://docs.opensafely.org/getting-started/) guide does describe how to run an OpenSAFELY study on Gitpod, which is an alternative.
